projects/plugins/jetpack/modules/gravatar-hovercards.php

```diff
@@ -200,19 +200,24 @@ function grofiles_get_avatar( $avatar, $author ) {
 				$cache_key   = 'gravatar_profile_' . $email_hash;
 
 				$response_body = wp_cache_get( $cache_key, $cache_group );
-				if ( false === $response_body ) {
-					$response = wp_remote_get( esc_url_raw( 'https://en.gravatar.com/' . $email_hash . '.json' ) );
-
+				if ( $response_body ) {
+					$profile      = $response_body->entry[0];
+					$display_name = $profile->displayName; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+				} else {
+					$response = wp_remote_get( esc_url_raw( 'https://www.gravatar.com/' . $email_hash . '.json' ) );
```

Comment:
I note that requesting `www.gravatar.com` redirects to the corresponding `en.gravatar.com`, possibly based on request headers. It might be worth keeping the "en" URL to skip having to follow the redirect.